examples/bpk-component-chip/examples.tsx

```diff
@@ -0,0 +1,309 @@
+/*
```

Comment:
This file changes:
- migrated to TS
- migrated class components to functional components
- added BpkIconChip examples